src/http.ts

```diff
@@ -1271,7 +1271,9 @@ export class LemmyHttp {
           "Content-Type": "application/json",
           ...this.headers,
         },
-        body: JSON.stringify(form),
+        body: JSON.stringify(form, (_, val) =>
+          typeof val === "bigint" ? val.toString() : val
+        ),
```

Comment:
See if this one works, seems cleaner: 

https://github.com/LemmyNet/lemmy/pull/new/prettier_check

If it doesn't, I'll merge what you have.

Reply:
Done.